queue_services/account-mailer/src/account_mailer/email_templates/govm_business_invitation_email.html

```diff
@@ -1,44 +1,57 @@
     [[style.html]]
 </head>
+
 <body>
-<table style="width: 100%;" cellspacing="0" cellpadding="0">
-    <tr>
-        <td style="padding-left: 20px; padding-right: 20px;">
-            <table style="max-width: 640px; margin: 0 auto; border: 0; border-collapse: collapse; border-radius: 6px; background-color: #ffffff;"
-                   cellspacing="0" cellpadding="0">
-                <tr>
-                    <td style="padding-top: 30px; padding-bottom: 30px;">
-                        [[logo.html]]
-                    </td>
-                </tr>
-                <tr>
-                    <td>
-                        <h1 style="margin: 0; line-height: 34px; letter-spacing: -1px; font-family: Verdana, Arial, sans-serif; font-size: 24px; font-weight: 700;">
-                            You've been invited to create BC Registries account.
-                        </h1>
-                        <p style="margin-top: 25px; margin-bottom: 0; line-height: 24px; font-family: Verdana, Arial, sans-serif; font-size: 16px;">
-                            BC Registries staff have invited you to create a Business Registry account. You are the account administrator for the account {{ account_name }}.<br/> To complete the account setup, click the button below and create your account. Then login to your account and manage your team members.
-                        </p>
-                        <div style="margin-top: 45px;">
-                            <!--[if mso]> <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="{{ context_url }}" style="height:55px; v-text-anchor:middle; width:150px;" arcsize="10%" stroke="f" fillcolor="#003366" alias="CTA1">Join Team<w:anchorlock/> <center> <![endif]-->
-                            <a style="display: inline-block; width: 150px; color: #ffffff; border: none; border-radius: 6px; background-color: #003366; line-height: 55px; text-align: center; text-decoration: none; font-size: 16px; font-weight: bold;"
-                               href="{{ context_url }}">Create Account</a>
-                            <!--[if mso]> </center> </v:roundrect> <![endif]-->
-                        </div>
-                    </td>
-                </tr>
-            </table>
-        </td>
-    </tr>
-</table>
+    <table style="width: 100%;" cellspacing="0" cellpadding="0">
+        <tr>
+            <td style="padding-left: 20px; padding-right: 20px;">
+                <table
+                    style="max-width: 640px; margin: 0 auto; border: 0; border-collapse: collapse; border-radius: 6px; background-color: #ffffff;"
+                    cellspacing="0" cellpadding="0">
+                    <tr>
+                        <td style="padding-top: 30px; padding-bottom: 30px;">
+                            [[logo.html]]
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <h1
```

Comment:
The spacing of these seem weird

Reply:
yeah I think the formatter messed it up. @MatthewCai2002 can you please make sure only your change is reflected here?
For example like 3 was added for no reason, same with extra space on line 8